src/Microsoft.ComponentDetection/Microsoft.ComponentDetection.csproj

```diff
@@ -1,9 +1,10 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
         <OutputType>Exe</OutputType>
         <RuntimeIdentifiers>win-x64;linux-x64;osx-x64</RuntimeIdentifiers>
         <RootProjectDirectory>$(MSBuildThisFileDirectory)..\..\</RootProjectDirectory>
+        <NoWarn>NU1505</NoWarn>
```

Comment:
nit: Can you add a comment linking to [this][1] issue, so we know why it was added, and we can remove it later? 

[1]: https://github.com/dotnet/sdk/issues/24747

Reply:
```suggestion
        <NoWarn>NU1505</NoWarn> <!-- https://github.com/dotnet/sdk/issues/24747 -->
```